tailwind.css

```diff
@@ -1,3 +1,9 @@
 @tailwind base;
+
+* {
+  scrollbar-width: thin;
+  scrollbar-color: gray rgb(239, 239, 239);
+}
+
```

Comment:
I am not sure if this will work because tailwind.css is generated and due to some limitations and quick decisions is also living in multiple files, please check these commands in package.json: `tailwind:react` and `tailwind:legacy`

I think we could just move these classNames into this select.component, WDYT?
Btw. this is not built in tailwind but you can use custom classNames with this:
```ts
[scrollbar-color:gray_rgb(239,_239,_239)] [scrollbar-width:thin]
```

I'd suggest to stick to hex colors tho:
```ts
[scrollbar-color:gray_#efefef] [scrollbar-width:thin]
```

I hope it works!

Reply:
thanks, had no idea it's possible 🙌